sdk/storage/Azure.Storage.Webjobs.Extensions.Common/src/Shared/Queues/IQueueProcessor.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.WebJobs.Host.Executors;
+
+namespace Microsoft.Azure.WebJobs.Host.Queues.Listeners
```

Comment:
Is this used?

Reply:
No that's remnant of initial approach. I'll remove this.